### ISSUE TITLE
Multi Select Dropdown : On reset form or changing to initial values , NVDA/Narrator is reading the place holder text in multi select dropdown.

### ISSUE THREAD

**Issue by vaishnavi-115** [microsoft/fluentui#19513]
Environment Information
Package version(s): "@fluentui/react": "^7.126.0",
Browser and OS versions: Chrome , windows 10
Please provide a reproduction of the bug in a codepen:
Actual behavior:
On reset form or clearing the values to initial values on button click, NVDA/Narrator is reading the place holder text in multi select dropdown.

Expected behavior:
On reset form or clearing the values to initial values on button click, NVDA/Narrator should not read the place holder text in multi select dropdown.

**Comment by vaishnavi-115** [comment]
Please find the sandbox link to repro the issue.
https://codesandbox.io/s/unruffled-jones-sqeeq?file=/src/MultiSelectExample.js

**Comment by gouttierre** [comment]
@vaishnavi-115  - Thanks for filing this issue with us and updating it with the codepen. To set expectation our team will have a look at the issue but only once capacity allows.
  
@bsunderhus -  Would you be able to confirm if this is a regression, or if this behavior is an issue?
cc @smhigley

**Comment by smhigley** [comment]
Looks like it's because we have `aria-live="off"`, which is ironically not supported, and defaulting to assertive. If we remove the live region attribute entirely, it doesn't read out.

**Comment by smhigley** [comment]
I've added a PR fixing this in v8, though it's worth noting the codepen is in v6, which has different live region attributes and would need a different PR (probably not just a cherry-pick).
